mne/beamformer/_lcmv.py

```diff
@@ -285,6 +330,12 @@ def lcmv(evoked, forward, noise_cov, data_cov, reg=0.05, label=None,
         detected automatically. If int, the rank is specified for the MEG
         channels. A dictionary with entries 'eeg' and/or 'meg' can be used
         to specify the rank for each modality.
+    whitening: False | True
+        whiten leadfield / covariance matrix
```

Comment:
I don't think you need this parameter. Maybe you can do if noise_cov is not None then don't whiten. However I am really skeptical that you can pool grad, mag and even eeg without using a noise covariance. You should test with combined sensors to assess this.